examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb

```diff
@@ -499,6 +499,7 @@
     "    sim = habitat_sim.Simulator(cfg)\n",
     "    # Managers of various Attributes templates\n",
     "    obj_attr_mgr = sim.get_object_template_manager()\n",
+    "    obj_attr_mgr.load_configs(str(os.path.join(data_path, \"objects\")))\n",
```

Comment:
This is put here to allow for loading the objects that was performed before when the physics manager attributes configuration was loaded. Calling this functionality will not be necessary once the DatasetAttributesManagers functionality is finished; the dataset_config.json file will hold, among other things, references to paths for the objects, stages, lighting setups, navmeshes, etc.